server/app.js

```diff
@@ -0,0 +1,40 @@
+const express = require('express');
+const path = require('path');
+const bodyParser = require('body-parser');
+const cookieParser = require('cookie-parser');
+// const router = require('./controllers');
+const { authCheck } = require('./authentication/authentication');
+
+const app = express();
+app.set('port', process.env.PORT || 4000);
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(express.static(path.join(__dirname, '..', 'public')));
+app.use(cookieParser());
+
+app.use((req, res, next) => {
+  authCheck(req, (authErr, token) => {
+    if (authErr) {
+      req.token = null;
+      req.userauthed = false;
+      next();
+    } else {
+      req.token = token;
+      req.userauthed = true;
+      if (token.role === 'admin') {
```

Comment:
why your role admin?

Reply:
this lines if we have different privileges, and now we work just on end user (girls)
so you think its right to check the role?